project.clj

```diff
@@ -8,15 +8,12 @@
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "0.6.1"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.8.1-SNAPSHOT"]
                    :inherit [:managed-dependencies]}
 
   :dependencies [[puppetlabs/pcp-common "1.1.4"]
 
-                 ;; Transitive dependencies on jetty for stylefuits/gniazdo
-                 ;; to use a stable jetty release (gniazdo specifies 9.3.0M1)
-                 [org.eclipse.jetty.websocket/websocket-client "9.2.10.v20150310"]
```

Comment:
Yeah, I ended up having to hard-code the dependency with a version to get `lein repl` behavior to work correctly. Which is not the worst, because we have to update anyway to pick up new clj-parent if we want to test a new jetty version.